packages/swarm-components/src/TextInput.jsx

```diff
@@ -55,7 +55,7 @@ export type TextInputProps = {
 	/**
 	 * max length of input field
 	 */
-	maxLength?: number,
+	maxLength?: ?number,
```

Comment:
Same thing here with the maybe types.  It's already optional so wondering why it would be necessary to explicitly accept `null` and `undefined` values.